Sources/AppcuesKit/Presentation/UI/AppcuesView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  AppcuesView.swift
+//  AppcuesKit
+//
+//  Created by James Ellis on 8/18/22.
+//  Copyright © 2022 Appcues. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+public class AppcuesView: UIView {
+
+    private var isConfigured = false        // appcues instance, embed ID, and view controller set
+    private var isAttachedToView = false    // didMoveToSuperView has been called
+    private var hasTracked = false          // has tracked the embed to check for qualified content
+
+    private var appcues: Appcues?
```

Comment:
I think this needs to be `weak`?

And I appreciate the explicitness of having `isConfigured: Bool`, but if `appcues` becomes weak, the `let appcues = appcues` check on L52 makes checking `isConfigured` redundant.

Reply:
good point! updated